src/main/scala/craft/JTAG.scala

```diff
@@ -20,5 +20,6 @@ trait JTAGTestLevelIO {
   val tdi        = Input(Bool())
   val tdo        = Output(Bool())
   val tclk       = Input(Bool())
+  val tdo_driven = Output(Bool())
```

Comment:
Yeah, there's an always block in the TestDriver that ticks on the core clock. You can add some wires and hook them up to the JTAG inputs/outputs. 

http://www.asic-world.com/verilog/vbehave2.html